public/tabs/chat/messages/message_action.tsx

```diff
@@ -171,23 +167,18 @@ export const MessageActions: React.FC<MessageActionsProps> = ({
     },
     trace: {
```

Comment:
Nit: Since this trace button won't be used anymore. why don't we remove this button config?

Reply:
I think the tract button is still used on the message bubble for chatbot
<img width="428" alt="Screenshot 2025-03-17 at 11 34 02" src="https://github.com/user-attachments/assets/f122877a-5526-475c-97f9-a987c5e8788b" />